src/main/scala/com/datastax/gatling/plugin/metrics/HistogramLogger.scala

```diff
@@ -6,309 +6,209 @@
 
 package com.datastax.gatling.plugin.metrics
 
-import java.io.{FileOutputStream, PrintStream}
-import java.nio.file.Paths
-import java.util.Collections
-import java.util.concurrent.{ConcurrentSkipListMap, TimeUnit}
+import java.io.{Closeable, FileOutputStream, PrintStream}
+import java.nio.file.{Path, Paths}
+import java.util.concurrent.ConcurrentSkipListMap
+import java.util.concurrent.TimeUnit.{MILLISECONDS, NANOSECONDS}
 
-import com.datastax.gatling.plugin.utils.ResponseTimers
-import com.fasterxml.jackson.databind.{ObjectMapper, SerializationFeature}
-import com.fasterxml.jackson.module.scala.DefaultScalaModule
-import com.typesafe.scalalogging.LazyLogging
+import com.datastax.gatling.plugin.utils.ResponseTime
+import com.typesafe.scalalogging.StrictLogging
 import io.gatling.core.Predef._
 import io.gatling.core.session.Session
 import org.HdrHistogram._
 
-import scala.collection.JavaConverters._
-import scala.collection.mutable.ArrayBuffer
-import scala.reflect.io.File
+import scala.collection.JavaConversions._
```

Comment:
This is the first usage of `JavaConversions` I'm seeing. Everywhere else we use `JavaConverters`, was this choice intentional?

Reply:
I can use `JavaConverters` as well, it only adds a few explicit `.asScala` method calls here and there